src/Microsoft.DocAsCode.Common/Loggers/ILoggerListener.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.DocAsCode.Common
 
     public interface ILoggerListener : IDisposable
     {
-        LogLevel LogLevelThreshold { get; set; }
+        LogLevel LogLevelThreshold { get; }
```

Comment:
>LogLevelThreshold [](start = 17, length = 17)

why do we need to expose this? From the code seems it's each listener to decide whether to record the log or not.